scripts/fund-pause-proxy.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -e
+
+CHANGELOG="0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F"
+NETWORK="goerli"
+VALUE="1ether"
+
+[[ "$(cast chain --rpc-url="$ETH_RPC_URL")" == "${NETWORK}" ]] || \
+  { echo "Please set a Goerli ETH_RPC_URL"; exit 1; }
+
+MCD_PAUSE_PROXY=$(cast call "${CHANGELOG}" \
+  'getAddress(bytes32)(address)' \
+  "$(cast --to-bytes32 "$(cast --from-ascii "MCD_PAUSE_PROXY")")")
```

Comment:
this can be simplified to: 
```suggestion
  "$(cast --format-bytes32-string "MCD_PAUSE_PROXY")
```